### PR TITLE
added more sub menus to the nav bar

### PR DESCRIPTION
**Pull request by Ann-Barnshaw**
Hi @lillian-busingye, 

Hope you are well.

Kindly review my code. 

Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -60,28 +60,125 @@
...
-                        <li class="sound-effects"><a href="#">Sound Effects</a></li>
-                        <li class="graphics-temps"><a href="#">Graphic Templates</a></li>
-                        <li class="graphics"><a href="#">Graphics</a></li>
-                        <li class="present-temps"><a href="#">Presentation Templates</a></li>
                         <li class="photos"><a href="#">Photos</a></li>
-                        <li class="fonts"><a href="#">Fonts</a></li>
-                        <li class="add-ons"><a href="#">Add ons</a></li>
-                        <li class="web-temp"><a href="#">Web Templates</a></li>
+                        <li class="fonts"><a href="#">Fonts</a>
+                        <div class="submenu-cont2a8">
```
Comment (lillian-busingye) [inline index.html]: Could you use a clear class name like `submenu-container`

File: index.html [hunk 2]
```diff
@@ -60,28 +60,125 @@
...
+                                <li><a href="#">Serif</a></li>
+                                <li><a href="#">Sans-Serif</a></li>
+                                <li><a href="#">Script and Handwritten</a></li>
+                                <li><a href="#">Decorative</a></li>
+                                <li><a href="#">All Fonts</a></li>
+                            </ul>
+                        </div>
+                        </li>
+                        <li class="add-ons"><a href="#">Add ons</a>
+                        <div class="submenu-cont2a9">
```
Comment (lillian-busingye) [inline index.html]: Could you use a definitive class name.

File: index.html [hunk 3]
```diff
@@ -60,28 +60,125 @@
...
+                                <li><a href="#">Adobe Photoshop</a></li>
+                                <li><a href="#">Adobe Lightroom</a></li>
+                                <li><a href="#">Adobe Illustrator</a></li>
+                                <li><a href="#">Procreate</a></li>
+                                <li><a href="#">All Add-ons</a></li>
+                            </ul>
+                        </div>
+                        </li>
+                        <li class="web-temp"><a href="#">Web Templates</a>
+                        <div class="submenu-cont2a10">
```
Comment (lillian-busingye) [inline index.html]: Could you use a definitive class name.

File: style.css [hunk 4]
```diff
@@ -343,33 +396,261 @@ input:focus {
...
-    cursor: pointer;
+.cont2_a li:hover .submenu-cont2a4 ul {
+    display: block;
+    text-align: left;
+}
+.cont2_a li:hover .submenu-cont2a4 ul li {
+    list-style: none;
+    margin: 25px 15px;
+}
+.cont2_a li:hover .submenu-cont2a4 ul li a:hover {
```
Comment (lillian-busingye) [inline style.css]: Could you use a definitive class name.

**Review by lillian-busingye — CHANGES_REQUESTED** [review]
Everything looks good to me except the class names please fix those.

File: index.html (hunk 1 above)
Comment (Ann-Barnshaw) [reply index.html]: Thank you @lillian-busingye. 

I have changed the class names.

Comment (Ann-Barnshaw) [reply index.html]: Thank you @lillian-busingye 

I have changed the class names

File: index.html (hunk 2 above)
Comment (Ann-Barnshaw) [reply index.html]: Thank you @lillian-busingye

I have changed the class names.

File: index.html (hunk 3 above)
Comment (Ann-Barnshaw) [reply index.html]: Thank you @lillian-busingye

I have changed the class names.

File: style.css (hunk 4 above)
Comment (Ann-Barnshaw) [reply style.css]: Thank you @lillian-busingye

I have changed the class names.

**Pull request merged**